src/main/java/carpet/script/api/WorldAccess.java

```diff
@@ -1797,4 +1760,62 @@ public static double sampleNoise(NoiseRouter router, ServerLevel level, String w
         }
         return DensityFunctions.zero();
     });
+
+    private static Object[] find_blockstate_extra_args(List<Value> lv, int start,BlockState sourceBlockState,String name_in_exception){
```

Comment:
name (camelCase), whitespaces, and return a typed Pair (at least)

Reply:
solved